server/index.js

```diff
@@ -13,10 +14,10 @@ app.use(function(req, res, next) {
 
 app.use('/', express.static(path.join(__dirname, '../public')));
 
-app.use('/rooms/:id', express.static(path.join(__dirname, '../public')));
+app.use('/gallery/:listingId', express.static(path.join(__dirname, '../public')));
```

Comment:
Same as previous comments:
```suggestion
app.use('/:listingId/gallery/', express.static(path.join(__dirname, '../public')));
```